.changeset/great-mayflies-shake.md

```diff
@@ -0,0 +1,22 @@
+---
+'@formspree/react': minor
+'@formspree/core': minor
```

Comment:
@colevscode we have some bug fixes in this PR but everything seems backward-compatible, so I think this is a minor bump.